com.unity.formats.fbx/Documentation~/api_index.md

```diff
@@ -1,16 +1,43 @@
-# Developer’s Guide
+---
+uid: api_index
+---
 
-As a developer, you have access to the FBX Exporter from C# scripting. You can use the basic API by providing a single GameObject or a list of GameObjects.
+# FBX Exporter scripting API
 
-## Managing export settings
+The FBX Exporter package includes an API that allows to create C# scripts and applications to handle FBX export processes based on your custom needs.
```

Comment:
I would say "write" or "allows you to" instead of just "create", because in a first read it feels the API it selfs creates C# scripts.